src/Components/TeamCard/TeamCard.js

```diff
@@ -1,10 +1,13 @@
 
 import "./TeamCard.css";
+import imageContainer from './../../utils/teamImages'
 
 const TeamCard = ({TeamCard_Details}) => {
+
   return (
     <div className="TeamCard">
-      {/* <img src={require('utils/images/yash.jpg')} /> */}
+      <img src={imageContainer[TeamCard_Details["Name"]]} />
```

Comment:
This approach looks good to me 😄

Reply:
Thank you @yashsehgal . I haven't madded any change into the package-lock.json . I guess when I used npm i it may have made some changes. Let me know if any bug appears.

Reply:
> Thank you @yashsehgal . I haven't made any change into the package-lock.json . I guess when I used npm i it may have made some changes. Let me know if any bug appears.

Oh, then it's fine - I will look into this.